### ISSUE TITLE
Stress test failure in DropwizardExportsTest

### ISSUE THREAD

**Issue by dhoard** [prometheus/client_java#944]
Stress test failure in DropwizardExportsTest

Expected

```
# TYPE hist summary
# HELP hist Generated from Dropwizard metric import (metric=hist, type=io.dropwizard.metrics5.Histogram)
hist{quantile="0.5"} 49.0
hist{quantile="0.75"} 74.0
hist{quantile="0.95"} 94.0
hist{quantile="0.98"} 97.0
hist{quantile="0.99"} 98.0
hist{quantile="0.999"} 99.0
hist_count 100
# EOF
```

Actual

```
# TYPE hist summary
# HELP hist Generated from Dropwizard metric import (metric=hist, type=io.dropwizard.metrics5.Histogram)
hist{quantile="0.5"} 50.0
hist{quantile="0.75"} 75.0
hist{quantile="0.95"} 95.0
hist{quantile="0.98"} 98.0
hist{quantile="0.99"} 99.0
hist{quantile="0.999"} 99.0
hist_count 100
# EOF
```

Stacktrace

```
org.junit.ComparisonFailure: 
	at org.junit.Assert.assertEquals(Assert.java:117)
	at org.junit.Assert.assertEquals(Assert.java:146)
	at io.prometheus.metrics.instrumentation.dropwizard5.DropwizardExportsTest.testHistogram(DropwizardExportsTest.java:167)
	at io.prometheus.metrics.instrumentation.dropwizard5.DropwizardExportsTest.stressTestHistogram(DropwizardExportsTest.java:138)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:232)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:55)
```

**Comment by dhoard** [comment]
Stress test

```
    @Test
    public void stressTestHistogram() {
        for (int i = 1; i <= 1000000; i++) {
            System.out.println("stressTestHistogram[" + i + "]");
            setUp();
            testHistogram();
        }
    }
```

**Comment by fstab** [comment]
Should be fixed with #948. Thanks for reporting!
